lightcurve_fitting/speccal.py

```diff
@@ -435,7 +435,7 @@ def calibrate_spectra(spectra, lc, filters=None, order=0, subtract_percentile=No
         return fig
 
 
-def create_wiserep_tsv(specpaths, wiserep_dir, verbose=False):
+def create_wiserep_tsv(specpaths, wiserep_dir, verbose=False, instruments=None):
```

Comment:
Can you add `instruments` to the docstring?

Reply:
done!